table.csv

```diff
@@ -557,6 +557,8 @@ eip-191,                        varsig,         0xd191,         draft,      EIP-
 jwk_jcs-pub,                    key,            0xeb51,         draft,      JSON object containing only the required members of a JWK (RFC 7518 and RFC 7517) representing the public key. Serialisation based on JCS (RFC 8785)
 fil-commitment-unsealed,        filecoin,       0xf101,         permanent,  Filecoin piece or sector data commitment merkle node/root (CommP & CommD)
 fil-commitment-sealed,          filecoin,       0xf102,         permanent,  Filecoin sector data commitment merkle node/root - sealed and replicated (CommR)
+dms3,                           multiaddr,      0x100000,       draft,      Decentralized Multi-State-Space Search
+dms3-key,                       ipld,           0x100001,       draft,      DMS3 Public Key
```

Comment:
```suggestion
dms3-key,                        key,           0x100001,       draft,      Decentralized Multi-State-Space Search Public Key
```

unfortunately you can't use `ipld` if this isn't a codec that can decode bytes into something that can contain _links_; if it's a codec that doesn't involve links then it's a `serialization`, but given you're saying it's a public key then I suspect it's not even that.

I'd suggest it's probably just a `key`, but you also may want to consider the name `dms3-pub` to match the other public key types. But this also raises the question: is this actually a key type? Can you just use one (or more) of the existing key types for this?

Reply:
this objection still stands, can't be `ipld` here